modules/tinymce/src/core/main/ts/mode/Readonly.ts

```diff
@@ -116,6 +116,9 @@ const registerReadOnlyContentFilters = (editor: Editor) => {
 
 const isClickEvent = (e: Event): e is MouseEvent => e.type === 'click';
 
+const allowedEvents: ReadonlyArray<string> = [ 'copy' ];
+
+const isReadOnlyAllowedEvent = (e: Event) => Arr.contains(allowedEvents, e.type);
```

Comment:
```suggestion
const isReadOnlyAllowedEvent = (e: Event) => Arr.contains(allowedEvents, e.type);

```
This still needs an empty line to break up the functions 😄